sagemaker-python-sdk/pytorch_horovod_mnist/pytorch_mnist_horovod.ipynb

```diff
@@ -71,7 +71,36 @@
    "metadata": {},
```

Comment:
<p>What is the second entry in tuple for? It's confusing if someone is reading through this for the first time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>It's the checksum key that is used to make sure that the data file is not corrupted. It is a requirement by the dataset.MNIST class. I will add in an explanation</p>

Reply:
<p>Thanks</p>